.circleci/config.yml

```diff
@@ -33,6 +33,10 @@ jobs:
           name: Run Core tests
           command: env PYTEST_ADDOPTS="--exitfirst" make test
 
+      - run:
+          name: Check NumPy typing against latest 3 minor versions
```

Comment:
[Checked on CI ](https://app.circleci.com/pipelines/github/openfisca/openfisca-core/318/workflows/53e5b229-6848-497a-ba84-eedca7008cd1/jobs/7005) and it's great ❤️